test/test_collections.py

```diff
@@ -168,6 +195,10 @@ def test_equal(self):
         self.assertEqual(self.d, b)
         c = [('a', 'foo, bar')]
         self.assertNotEqual(self.d, c)
+        
+        # For some reason, the test above doesn't run into the __eq__ function,
+        # lacking test coverage for line 165, so comparing directly here
+        self.assertFalse(self.d == 2)
```

Comment:
Without that line, the testrun reported missing coverage for _collections#163, and I had expected, this not equal comparison the line above would also call __eq__ finally, but for some reason it didn't.